tools/metricsdocs/metricsdocs.go

```diff
@@ -5,7 +5,7 @@ import (
 	"sort"
 
 	"github.com/kubevirt/cluster-network-addons-operator/pkg/monitoring"
-	"github.com/kubevirt/cluster-network-addons-operator/tools/metrics-parser"
+	metricsparser "github.com/kubevirt/cluster-network-addons-operator/tools/metrics-parser"
```

Comment:
should we disable fmt / vet alerts on this ?
because it is nice to have, not mandatory
maybe it suggest it here because the `-` which makes the two names not the same

Reply:
I don't to keep this level of fmt, fmt adds it automatically anyway, it's not like I need to do anything.
But we can certainly examine further how much we want fmt to alert us in a separate PR.